### PR TITLE
links to forums

### PR DESCRIPTION
**Pull request by swomma**

### CODE REVIEW

File: app/scripts/views/unit_awols.js [hunk 1]
```diff
@@ -23,6 +23,7 @@ define([
             this.collection.setFilter("days", days).fetch({reset: true}); // TODO: Add error handling, loading indicator?
         },
         serializeData: function() {
+            alert(this.collection.models);
```
Comment (wilson29thid) [inline app/scripts/views/unit_awols.js]: Eh?

File: app/scripts/helpers/custom.js [hunk 2]
```diff
@@ -159,6 +159,15 @@ define([
         return str.replace(from, to);
     });
 
+    Handlebars.registerHelper('link_format', function (forum_object, forum_id, linkType, replacer ) {
+        var link = forum_object[forum_id][linkType];
+        return link.replace('%s', replacer);
+    });
+
+    Handlebars.registerHelper('json', function(context) {
```
Comment (wilson29thid) [inline app/scripts/helpers/custom.js]: Do you use this anywhere?

Comment (swomma) [inline app/scripts/helpers/custom.js]: in service record forum links for now

File: app/scripts/views/unit_awols.js (hunk 1 above)
Comment (swomma) [inline app/scripts/views/unit_awols.js]: My bad, dev stuff, should be removed

**Comment by wilson29thid** [comment]
Merged manually

**Pull request closed without merging**